openwisp_radius/urls.py

```diff
@@ -13,17 +14,22 @@ def get_urls(api_views=None, social_views=None):
     """
     if not social_views:
         from .social import views as social_views
-    urls = [url(r'api/v1/', include(get_api_urls(api_views)))]
+    urls = [path('api/v1/', include(get_api_urls(api_views)))]
     if app_settings.SOCIAL_LOGIN_ENABLED:
         urls.append(
-            url(
-                r'^freeradius/social-login/(?P<slug>[\w-]+)/$',
+            path(
+                'freeradius/social-login/<slug>/',
                 social_views.redirect_cp,
                 name='redirect_cp',
             )
         )
     return urls
 
 
-app_name = 'radius'
-urlpatterns = get_urls()
```

Comment:
This is a backwards incompatible change, as a different output is given by the file, but I don't think it's used in any dependency.